test/gui/tst_sharing/test.feature

```diff
@@ -168,3 +168,53 @@ Feature: Sharing
             | expireDate | 2038-07-21 |
         Then the fields of the last public link share response of user "Alice" should include on the server
             | expireDate | 2038-07-21 |
+
+
+	Scenario: simple sharing of a folder by public link without password
+        Given user "Alice" has been created on the server with default attributes
+        And user "Alice" has set up a client with these settings and password "1234":
+            """
+            [Accounts]
+            0\Folders\1\ignoreHiddenFiles=true
+            0\Folders\1\localPath=%client_sync_path%
+            0\Folders\1\paused=false
+            0\Folders\1\targetPath=/
+            0\Folders\1\version=2
+            0\Folders\1\virtualFilesMode=off
+            0\dav_user=alice
+            0\display-name=Alice
+            0\http_oauth=false
+            0\http_user=alice
+            0\url=%local_server%
+            0\user=Alice
+            0\version=1
+            version=2
+            """
```

Comment:
this is an unrelated issue but can't we hide this? only variable here is `dav_user` and `display_name`

Reply:
Maybe we can do that in next PR. @individual-it What do you think about this?

Reply:
sounds good, please open an issue and plan it